salt/control/virt.sls

```diff
@@ -67,6 +67,14 @@ salt_control_virt_{{ cluster_name }}_{{ node_name }}:
 #  - path: /srv/salt-images/{{ node_name }}.{{ cluster.domain }}/system.qcow2
 #  - id_: {{ node_name }}.{{ cluster.domain }}
 #  - unless: virsh list | grep {{ node_name }}.{{ cluster.domain }}
+
+salt_virt_autostart_{{ cluster_name }}_{{ node_name }}:
```

Comment:
It should be also controlled by some parameter, eg. `node.get('autostart', True)`